common.mk

```diff
@@ -190,11 +190,11 @@ test-coverage:  ## Generate coverage report
 
 .PHONY: docs
 docs:  ## Build documentation
-	uv run --extra docs sphinx-build -b html -W $(DOCS) $(DOCS)/_build
+	uv run --extra docs sphinx-build -b html $(DOCS) $(DOCS)/_build
 
 .PHONY: docs-auto
 docs-auto:  ## Build and host docs with sphinx-autobuild
-	uv run --extra docs sphinx-autobuild -b html --open-browser --port=8080 --watch $(PROJECT) -W $(DOCS) $(DOCS)/_build
+	uv run --extra docs sphinx-autobuild -b html --open-browser --port=8080 --ignore *.kate-swp --ignore docs/reference/commands/** --watch $(PROJECT) $(DOCS) $(DOCS)/_build
```

Comment:
I'm wondering if we could pipe the exclusion list from `docs/conf.py` into the arguments here.

Reply:
Not without extra work that's out of scope here.

Ideally we'd get [this fixed upstream](https://github.com/sphinx-doc/sphinx-autobuild/issues/140). Likewise, it'd be nice to get [this fixed too](https://github.com/sphinx-contrib/sphinx-lint/issues/66).

Reply:
As I should have guessed, you two have already flagged this. :)